src/main/java/net/fabricmc/loom/configuration/CompileConfiguration.java

```diff
@@ -155,7 +156,11 @@ private synchronized void setupMinecraft(ConfigContext configContext) throws Exc
 
 		final MinecraftMetadataProvider metadataProvider = MinecraftMetadataProvider.create(configContext);
 
-		final var jarConfiguration = extension.getMinecraftJarConfiguration().get();
+		var jarConfiguration = extension.getMinecraftJarConfiguration().get();
+
+		if (jarConfiguration == MinecraftJarConfiguration.MERGED && !metadataProvider.getVersionMeta().isVersionOrNewer("2012-07-25T22:00:00+00:00" /* 1.3 release date */)) {
```

Comment:
NIT: Maybe move the release time to `Constants`? Will save duplicating it everywhere.